lib/adagrams.rb

```diff
@@ -0,0 +1,145 @@
+require 'pry'
+require 'csv'
+
+# WAVE 1
+# The draw_letters method starts with a fresh pool of tiles and returns
+# an array of 10 letters (the "hand").
+def draw_letters
```

Comment:
Small issue here, each letter has an **equal** chance to be drawn.  Instead an A should be 9 times more likely than a J.

Instead you'll need an Array like:
```ruby
["A","A","A","A","A","A","A","A","A","B","B"...]
```
And draw from that array, removing letters as they're drawn.